docs/reference/indices/flush.asciidoc

```diff
@@ -122,26 +126,25 @@ which returns something similar to:
 // TESTRESPONSE[s/"retention_leases" : "primary_term:1;version:1;id:replica-0;retaining_seq_no:0;timestamp:1547235588;source:replica"/"retention_leases": $body.indices.twitter.shards.0.0.commit.user_data.retention_leases/]
 <1> the `sync id` marker
 
+NOTE: The `sync_id` marker is removed as soon as the shard is flushed again,
+and {es} may trigger a flush at any time. In practice this means that one
+should consider any indexing operation on an index as having removed the
+`sync_id` markers.
+
 [float]
 === Synced Flush API
 
-The Synced Flush API allows an administrator to initiate a synced flush manually. This can be particularly useful for
-a planned (rolling) cluster restart where you can stop indexing and don't want to wait the default 5 minutes for
-idle indices to be sync-flushed automatically.
-
-While handy, there are a couple of caveats for this API:
-
-1. Synced flush is a best effort operation. Any ongoing indexing operations will cause
-the synced flush to fail on that shard. This means that some shards may be synced flushed while others aren't. See below for more.
-2. The `sync_id` marker is removed as soon as the shard is flushed again. That is because a flush replaces the low level
-lucene commit point where the marker is stored. Uncommitted operations in the transaction log do not remove the marker.
-In practice, one should consider any indexing operation on an index as removing the marker as a flush can be triggered by Elasticsearch
-at any time.
-
-
-NOTE: It is harmless to request a synced flush while there is ongoing indexing. Shards that are idle will succeed and shards
- that are not will fail. Any shards that succeeded will have faster recovery times.
+The Synced Flush API allows an administrator to initiate a synced flush
```

Comment:
Why did you change the old explanation?

Reply:
The changes I made were to address a number of points that I originally found confusing:

- It says it's useful specifically for rolling restarts, but it's actually useful for all restarts whether rolling or otherwise.
- The note about future flushes removing the `sync_id` is out of place in this section on the API, since it applies even if you don't use this API.
- It introduces the concept of a "low level Lucene commit point" without definition or explanation. This isn't really necessary, we can make that same point more clearly by talking about flushes directly.
- The formatting (i.e. the numbered list and the `NOTE:` callout) were unnecessary and made it harder to read than just straight prose.